### PR TITLE
PROD - Calculation fix for TAL-77

### PR DESCRIPTION
**Pull request by jmgasper**
https://topcoder.atlassian.net/browse/TAL-77

### CODE REVIEW

**Review by Aranyajit — APPROVED** [review]

File: src/services/SearchService.js [hunk 1]
```diff
@@ -188,11 +188,22 @@ async function addSkillScore(results, query){
       }
       item.skillScore = Math.round(score / query.skillIds.length * 100) / 100
 
-      if(!item.availableForGigs){
+      if(item.isAvailableForGigs === null){
```
Comment (testflyjets) [inline src/services/SearchService.js:191]: Are we sure we want to use `===` here and not `==`?

**Review by testflyjets — APPROVED** [review]
Confirm the equality check is what we want.

File: src/services/SearchService.js (hunk 1 above)
Comment (jmgasper) [reply src/services/SearchService.js:191]: No, good catch, thanks.

Comment (jmgasper) [reply src/services/SearchService.js]: Fixed now

**Pull request merged**